src/main/java/space/obminyashka/items_exchange/controller/request/SubcategorySearch.java

```diff
@@ -0,0 +1,21 @@
+package space.obminyashka.items_exchange.controller.request;
+
+import io.swagger.v3.oas.annotations.Parameter;
+import jakarta.validation.constraints.Positive;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+public class SubcategorySearch {
```

Comment:
I was mistaken in saying that DTO is for search but for Filtering.  Kindly ask you to update the name.